pastel/pastel.go

```diff
@@ -33,4 +33,10 @@ type Client interface {
 	// Verify verifies signed data by the given its signature and pastelID, if successful returns true.
 	// Command `pastelid verify "text" "signature" "PastelID"`.
 	Verify(ctx context.Context, data []byte, signature, pastelID string) (ok bool, err error)
+	// ActTickets returns activated art tickets.
+	// Command `tickets list act`.
+	ActTickets(ctx context.Context, actType ActTicketType, minHeight int) (ActTickets, error)
+	// ActTickets returns art registration tickets.
```

Comment:
And the comment here is not ActTickets

Reply:
Thanks for pointing out. Comment is ActTickets. Pls see line # 36
As you'd know, this is normal practice in Go when comments are muti-line